Common/src/toolboxes/MMS/CInviscidVortexSolution.cpp

```diff
@@ -77,12 +77,14 @@ CInviscidVortexSolution::CInviscidVortexSolution(unsigned short val_nDim,
     SU2_MPI::Error("Unsteady mode must be selected for the inviscid vortex",
                    CURRENT_FUNCTION);
 
-  if(config->GetKind_Regime() != COMPRESSIBLE)
+  if(Kind_Solver != EULER && Kind_Solver != NAVIER_STOKES && Kind_Solver != RANS &&
```

Comment:
This one is not solver specific

Reply:
Yeah it is. It only works and is intended for the compressible flow solver. What happens if the kind solver is not a flow solver ? I still think that this way is much more obvious for which solvers this part of the code is intended (and tested) for.

Reply:
Ok, we really need some more expressive and encapsulated way to express these things then.